### DISCUSSION TITLE
ng-matero-settings

### DISCUSSION THREAD

**Discussion by rahulpd101** [ng-matero/ng-matero#580] [Q&A] [answered]
The issue I am facing is, I am not able to login in my application even after authenticating with cookie. When I looked up in the original application template, I found out that there's an '**ng-matero-settings**' key in the local storage which is not loading in my case.

How is ng-matero-settings appended in localStorage with all the object values?
I don't see any direct append methods to localStorage. There are methods in **settings.service.ts** but they don't seem to append the 'ng-matero-settings' in local storage.
![settings](https://github.com/ng-matero/ng-matero/assets/37538741/23e05b45-676d-4e3d-b6f4-9d4223856630)

**Comment by nzbin** [comment, marked as answer]
It doesn't matter whether `ng-matero-settings` in local storage.
https://github.com/ng-matero/ng-matero/blob/2a5759f277e04c451dfb5292e00b5a2ede813436/src/app/core/bootstrap/settings.service.ts#L57-L61

The authentication of ng-matero use token by default. If you use cookie, you should implement your own logic with the interceptor.
https://github.com/ng-matero/ng-matero/blob/2a5759f277e04c451dfb5292e00b5a2ede813436/src/app/core/interceptors/token-interceptor.ts#L39-L42
